easybuild/easyconfigs/p/Python/Python-3.7.2-GCCcore-8.2.0.eb

```diff
@@ -42,6 +42,7 @@ installopts = " && ln -s %(installdir)s/bin/pip3 %(installdir)s/bin/pip"
 exts_default_options = {
```

Comment:
missing `'source_urls': [PYPI_SOURCE]`, if they are not specified for each extension

Reply:
Thanks. I added it here and removed it from the dict below to unify with the other EC